illumos-utils/src/zpool.rs

```diff
@@ -213,6 +213,16 @@ impl Zpool {
         }
     }
 
+    pub fn export(name: &ZpoolName) -> Result<(), Error> {
+        let mut cmd = std::process::Command::new(PFEXEC);
+        cmd.env_clear();
+        cmd.env("LC_ALL", "C.UTF-8");
+        cmd.arg(ZPOOL).arg("export").arg(&name.to_string());
+        execute(&mut cmd).map_err(Error::from)?;
```

Comment:
`.map_err(Error::from)` shouldn't be required, `?` does that already I think.

Reply:
Nice catch, copy-pasted from a method where it was required. Fixed in 5623fe1d